packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -137,8 +147,30 @@ def add_element(
     ):
         # TODO: validate that element "name" is unique
         # make a container that adds margin around the element
-        container = Container(margin=5, content=element)
-        self._elements.visible.append(container)
+
+        # if it's a Container we don't create our own margin container to allow setting
+        # of padding and absolute position manually by user
+        if isinstance(element, Container):
+            new_element = element
+        else:
+            new_element = Container(margin=5, content=element)
```

Comment:
maybe good to document that all elements will have a margin of 5 unless the user creates the containers himself?

Reply:
Yeah that's already been the behaviour since 1.<something>, I thought I documented it in the release notes but cannot find it.

I guess the question is that should we document this around the Open Container keyword or at the top level library documentation? I was actually considering adding a sub-heading into the docs like what playwright lib does here:
<img width="851" alt="Screenshot 2023-03-08 at 10 21 51" src="https://user-images.githubusercontent.com/13387304/223660103-089fc5a3-e35d-4a0f-bffb-7a1d333d2366.png">

talking about layouting in general. That would fix this issue.

Reply:
Documented in 0525e0dc